package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "pebble-timer",
+  "name": "ripple",
   "author": "conor.code@gmail.com",
   "version": "1.0.0",
   "keywords": ["pebble-app"],
   "private": true,
```

Comment:
this is likely going to need to change when we publish, but that's covered in #5